ui/src/flux/components/FluxScriptEditor.tsx

```diff
@@ -111,6 +111,17 @@ class FluxScriptEditor extends PureComponent<Props, State> {
     )
   }
 
+  private handleMouseEnter = (e: MouseEvent<HTMLDivElement>) => {
+    const {width, height} = e.currentTarget.getBoundingClientRect()
+    const {width: prevWidth, height: prevHeight} = this.containerDimensions
```

Comment:
do you need to use a lodash get or something here or give a default to this.containerDimensions? the first time the mouse enters, wont this.containerDimensions be undefined?